src/entities/college.entity.ts

```diff
@@ -0,0 +1,19 @@
+import { Entity, PrimaryGeneratedColumn, Column } from "typeorm";
+
+@Entity()
+export class College {
+  @PrimaryGeneratedColumn()
+  id: number;
```

Comment:
Your entity file is to bare bones

Nothing showing if the data/column is nullable or not (? !), no data type as well